src/egon/data/datasets/scenario_path/import_status2019.py

```diff
@@ -0,0 +1,87 @@
+"""
+Read eTraGo tables for the status2019 and import it to db
+"""
+import os
+import subprocess
+
+import pandas as pd
+
+from egon.data import config, db
+from egon.data.datasets import Dataset
+
+
+class Import_Status2019(Dataset):
+    def __init__(self, dependencies):
+        super().__init__(
+            name="import_status2019",
+            version="0.0.1",
+            dependencies=dependencies,
+            tasks=(import_scn_status2019,),
+        )
+
+
+def import_scn_status2019():
+    """
+    Read and import the scenario status2019 and import it into db
+
+    Parameters
+    ----------
+    *No parameters required
+
+    """
+    # Connect to the data base
+    con = db.engine()
+
+    # Clean existing data for status2019
+    tables = pd.read_sql(
+        """
+        SELECT tablename FROM pg_catalog.pg_tables
+        WHERE schemaname = 'grid'
+        """,
+        con,
+    )
+
+    tables = tables[
+        ~tables["tablename"].isin(
+            [
+                "egon_etrago_carrier",
+                "egon_etrago_temp_resolution",
+            ]
+        )
+    ]
+
+    for table in tables["tablename"]:
+        db.execute_sql(
+            f"""
+        DELETE FROM grid.{table} WHERE scn_name = 'status2019';
+        """
+        )
+
+    my_env = os.environ.copy()
+    my_env["PGPASSWORD"] = "data"
```

Comment:
Is there an option to get the password from somewhere else?

Reply:
Now we can retrieve the password from datasets.yml a83ca47a3b6db8fdc277e366b76f8e3b9044f49d